### PR TITLE
[nrf fromlist] manifest: hal_nordic update

### PR DESCRIPTION
**Pull request by JuhaKortesalmi**
Upstream PR #: 80657

### CODE REVIEW

**Review by carlescufi — CHANGES_REQUESTED** [review]
We cannot pull a change that points to a PR

**Review by carlescufi — APPROVED** [review]

**Comment by JuhaKortesalmi** [comment]
> > We cannot pull a change that points to a PR
> 
> We also cannot point to an unmerged SHA. PR [zephyrproject-rtos/hal_nordic#239](https://github.com/zephyrproject-rtos/hal_nordic/pull/239) needs to be merged first.

So there is conflict between comments, which one I choose, full SHA or "pull/PR/head"?

**Comment by 57300** [comment]
> > > We cannot pull a change that points to a PR
> > 
> > 
> > We also cannot point to an unmerged SHA. PR [zephyrproject-rtos/hal_nordic#239](https://github.com/zephyrproject-rtos/hal_nordic/pull/239) needs to be merged first.
> 
> So there is conflict between comments, which one I choose, full SHA or "pull/PR/head"?

You should use pull/#/head while the hal_nordic PR is still open. Only update to SHA after it's merged.
Right now you're using an "impostor" SHA from your own fork.

What @carlescufi meant is that we cannot merge this sdk-zephyr PR until the hal_nordic PR is merged.

**Pull request closed without merging**
